gradle-suppressible-error-prone/src/main/java/com/palantir/gradle/suppressibleerrorprone/SuppressibleErrorPronePlugin.java

```diff
@@ -75,15 +77,18 @@ private void applyToJavaProject(Project project) {
                 .orElseThrow(
                         () -> new RuntimeException("SuppressibleErrorPronePlugin implementation version not found"));
 
-        // When auto-suppressing, there are two stages:
-        // 1. The first runs a bytecode patched version of errorprone (via an
-        //    artifact transform) that intercepts every error from every check and adds a custom fix, a
-        //    @RepeatableSuppressWarnings annotation to the relevant statement/method/field/class.
-        // 2. The second stage runs a single errorprone check: SuppressWarningsCoalesce, which will combine all
-        //    the @RepeatableSuppressWarnings and @SuppressWarnings annotations into one normal @SuppressWarnings
-        //    annotation.
-
-        setupErrorProneArtifactTransform(project);
+        // Don't need to consider the custom suppression logic if we're going to remove the suppressions anyway
+        if (!isRemovingSuppressions(project)) {
+            // When auto-suppressing, there are two stages:
+            // 1. The first runs a bytecode patched version of errorprone (via an
+            //    artifact transform) that intercepts every error from every check and adds a custom fix, a
+            //    @RepeatableSuppressWarnings annotation to the relevant statement/method/field/class.
+            // 2. The second stage runs a single errorprone check: SuppressWarningsCoalesce, which will combine all
+            //    the @RepeatableSuppressWarnings and @SuppressWarnings annotations into one normal @SuppressWarnings
+            //    annotation.
+
+            setupErrorProneArtifactTransform(project);
+        }
```

Comment:
I _think_ this should be fine, but curious if you can think of problems this might cause